data-science-onramp/feature-engineering/feature_engineering.ipynb

```diff
@@ -1,4795 +1,1019 @@
 {
```

Comment:
<p>Nits:</p><p> </p><p>Proposed:</p><p>The following cell will combine both dataframes for trips started and ended at the stations. Then, <strong>all</strong> NaN values will be <strong>replaced with a 0 </strong>since this indicates that no trips started or ended at <strong>that particular station</strong>. Lastly, the dataset will be cleaned by renaming the columns and converting the values to integers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahecole/python-docs-samples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Ping</p>

Reply:
<p>thank you thank you</p>

Reply:
<p>done</p>